eng/ReferenceAssemblies.props

```diff
@@ -8,6 +8,7 @@
 
   <PropertyGroup Condition="'$(IsReferenceAssembly)'=='true'">
     <OutputPath>$([MSBuild]::NormalizeDirectory('$(ReferenceAssemblyOutputPath)', '$(MSBuildProjectName)', '$(Configuration)'))</OutputPath>
+    <BaseIntermediateOutputPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', 'ref', '$(MSBuildProjectName)'))</BaseIntermediateOutputPath>
     <IntermediateOutputPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', 'ref', '$(MSBuildProjectName)', '$(Configuration)'))</IntermediateOutputPath>
```

Comment:
Do you know if this lets you remove IntermediateOutputPath and let that be derived from BaseIntermediateOutputPath?  Run a PP and see if the ordering allows.

Reply:
I just checked and we still need to set both as the changes in ProjectLayout.props in Arcade happen earlier: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectLayout.props#L17-L19